### DISCUSSION TITLE
netclient-mipsle - What is the correct target platform (router)?

### DISCUSSION THREAD

**Discussion by cmonty14** [gravitl/netmaker#2354] [Q&A]
Hi,

in the release section you're providing this asset:
[netclient-mipsle](https://github.com/gravitl/netmaker/releases/download/v0.14.3/netclient-mipsle)

What is the target platform for this package?
Can this be used in OpenWrt ramips target (with [this](https://openwrt.org/toh/mikrotik/rb760igs) router)?

THX

**Comment by pete1019** [comment]
That is a good question. Anyone knows the answer?

**Comment by shan100github** [comment]
It's good to follow https://docs.netmaker.org/advanced-client-install.html#notes-on-openwrt to install in OpenWrt device.
If it didn't workout share more details referring https://docs.netmaker.org/troubleshoot.html#netclient
